client_app/src/components/shelter/PastVolunteers.js

```diff
@@ -1,22 +1,30 @@
-import React, { Component } from "react";
 import AccountCircleIcon from "@mui/icons-material/AccountCircle";
 import NotificationsNoneIcon from "@mui/icons-material/NotificationsNone";
 import "../../styles/shelter/PastVolunteers.css";
 
-class PastVolunteers extends Component {
-  render() {
-    const { shiftDetails } = this.props;
+const PastVolunteers = ({shiftDetails}) => {
+    const currentTime = Date.now();
+    const filteredShifts = shiftDetails.filter(item => item.start_time < currentTime);
```

Comment:
I like this filter, i think we might need to eventually incorporate some sort of edge case if the timezone differs unless they are sorting by location

Reply:
We use Unix epoch for the start_time and end_time, so it won't be affected by the user's time zone.